packages/blocks/src/database-block/common/columns/rich-text/cell-renderer.ts

```diff
@@ -340,7 +340,7 @@ export class RichTextCellEditing extends BaseCellRenderer<Text> {
 
       this.column.captureSync();
       const text = new Text(this.inlineEditor.yText);
-      text.replace(inlineRange.index, length, '\n');
+      text.replace(inlineRange.index, 0, '\n');
```

Comment:
The situation in the video should have a length of 0, so it may not be the source of the error.